charts/elasticsearch/templates/client/es-client-deployment.yaml

```diff
@@ -87,10 +87,7 @@ spec:
       - name: es-client
         securityContext:
 {{ toYaml .Values.securityContext | indent 10 }}
-          capabilities:
-            add:
-              - IPC_LOCK
-              - SYS_RESOURCE
+{{ toYaml .Values.securityContext_esclient | indent 10 }}
```

Comment:
What is the behavior here if there are conflicting `capabilities` defined in `Values.securityContext` and `Values.securityContext_esclient` ?

Reply:
I don't like how it ends up defining the same keys multiple times. We should find a way to merge the second one into the first one.

Reply:
@shmanu017 actually i think this is pretty straightforward. we can just do this as an `if ... else` block, saying `if Values.securityContext_esclient` use those, otherwise use `Values.securityContext`. That will keep it from double-defining that block.